templates/perfalert_inactive_regression_nag_needinfo.txt

```diff
@@ -0,0 +1,5 @@
+:{{ nickname }}, since you are the author of the regressor, bug {{ extra[bugid]["regressor_id"] }}, which triggered this performance alert, could you please provide a progress update?
+
+If you need additional information/help, please needinfo the performance sheriff who filed this alert (they can be found in comment #0), or reach out in [#perftest](https://matrix.to/#/#perftest:mozilla.org), or [#perfsheriffs](https://matrix.to/#/#perfsheriffs:mozilla.org) on Element.
```

Comment:
> needinfo the performance sheriff who filed this alert 

Is it the same person who files the bug? If so, we could mention their nickname instead.

Reply:
Unfortunately no, it's the intermittent bugs filer that is the reporter and the sheriff is mentioned in comment #0.